.github/workflows/autobuild.yml

```diff
@@ -14,21 +13,19 @@
 #     for every tag that starts with 'r' and does not have any suffix:
 #          - do CodeQl while building for every platform
 #          - publish the created binaries/packs only as artifacts/appendix as a release
-#
-
 
 on:
   workflow_dispatch:
   push:
     tags:
       - "r*"
     branches:
-      - "autobuild*"  # for developers: branches starting with autobuild will be built and evaluated on each push
-      - "autobuild/*" # for developers: branches starting with autobuild will be built and evaluated on each push
+      # For developers: Branches starting with autobuild will be built and evaluated on each push.
+      - "autobuild**"
       - "master"
-    #  - "*"
-  pull_request: # The branches below must be a subset of the branches in "push"
+  pull_request:
     branches:
+      # These branches must be a subset of the branches in "push":
       - master
 
 name:                               Auto-Build
```

Comment:
Below at create_release:

Check if we are doing a release or just a normal build. This must be done before actually building the app to clarify where to upload the binaries and if we need to create a release.

Reply:
Hm, I'd prefer improving the name instead. What about this:

```diff
- name:                          Prepare Auto-Build/Release
+ name:                          Create Github release if this is a release build
```

Reply:
Looking at the workflow, I think this name is slightly misleading. This step happens first, before the builds, so it isn't actually creating the Github release, just setting up variables to tell later steps to generate one. So "Prepare" or "Setup" is better, but happy for the rest of the name to be made clearer.